rush.json

```diff
@@ -4,14 +4,15 @@
  */
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush.schema.json",
-  "rushVersion": "5.99.0",
-  "pnpmVersion": "6.32.9",
+  "rushVersion": "5.100.1",
+  "pnpmVersion": "8.6.2",
   "ensureConsistentVersions": true,
   /**
    * Options that are only used when the PNPM package manager is selected
    */
   "pnpmOptions": {
-    "strictPeerDependencies": false
+    "strictPeerDependencies": false,
```

Comment:
In `azure-sdk-for-js`, we set `strictPeerDependencies: true`.  Have you tried enabling this setting in this repo?  If not, I think we should try in a separate PR after this one.

Reply:
I think there was a reason maybe it was because we needed a mist match of react version for a while but I don’t think we do anymore so might be a good cleanup